dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -86,3 +89,12 @@ includePathInWASReqURL.desc=Setting the Path parameter can allow the client/brow
 
 trackLoggedOutSSOCookies=Track logged out LTPA single signon tokens
 trackLoggedOutSSOCookies.desc=Specifies whether to track LTPA single signon tokens that are logged out on a server so that it can not be reused on the same server.
+
+overrideHttpAuthenticationMechanism=Name of HttpAuthenticationMechanism to override
+overrideHttpAuthenticationMechanism.desc=Specifies the HttpAuthenticationMechanism for overriding an application-defined HttpAuthenticationMechanism. The HttpAutheMechanism is defined by the Java EE 8 Security specification. The acceptable value is BASIC or FORM. When FORM is used, specify the loginFormURL and loginErrorURL properties.
+
+loginFormContextRoot=Context root for the login form
+loginFormContextRoot.desc=Specifies the context root of a form login page which is specified by loginFormURL. If this value is not set, the first path element of the loginFormURL is used as a context root. This value is applicable for Java EE 8 Security only.
+
+httpBasicAuthMechanismRealmName=Realm name for the HTTP basic authentication
+httpBasicAuthMechanismRealmName.desc=Specifies a realm name for Java EE 8 Security HTTP basic authentication. This value is applicable when overrideHttpAuthenticationMechanism is set as BASIC. The default value is defaultRealm.
```

Comment:
Just a couple minor things:

Could we mention what "HttpAuthenticationMechanism" is?
Also, I'd say "set to BASIC" instead of "as." :) Thanks!

Reply:
An explanation of HttpAuthenticationMechanism is in overrideHttpAuthenticationMechanism.desc and this attribute also refers it, so I didn't repeat the same text. Do you think it's better to add it?
Also, changed to "set to". Thank you Sarah.

Reply:
Oh, we don't need to add the entire description again. I just wanted to add one word to describe it, such as "HttpAuthenticationMechanism property" or "HttpAuthenticationMechanism attribute"...whatever word that you think best describes it. This is useful for translation because if "HttpAuthenticationMechanism" isn't translated, the users who speak different languages will still at least know that "HttpAuthenticationMechanism" is a property or an attribute or whatever it is. :)